media-gfx/openvdb/files/openvdb-4.0.2-fix-const-correctness-for-unittest.patch

```diff
@@ -0,0 +1,12 @@
+diff -Naur a/openvdb/unittest/TestFile.cc b/openvdb/unittest/TestFile.cc
```

Comment:
See patch comments above.

Reply:
I have also included more details for this patch, and added checks to ensure that it works for blosc > 1.15 and earlier. This patch was included in openvdb-7, but is necessary for earlier versions of openvdb to compile against newer versions of blosc.